src/schema/v2/schema.ts

```diff
@@ -394,6 +395,7 @@ export default new GraphQLSchema({
       updateUser: updateUserMutation,
       updateUserSaleProfile: updateUserSaleProfileMutation,
       updateQuiz: updateQuizMutation,
+      manageArtworksCollections: manageArtworksCollectionsMutation,
```

Comment:
**suggestion:** Maybe we can keep this list alphabetized, so move this up? #minor